browser/resources/brave_new_tab_page_refresh/components/app.style.ts

```diff
@@ -107,5 +138,47 @@ global.css`
     p {
       margin: 0;
     }
+
+    dialog, [popover] {
+      border: none;
+      color: inherit;
+      margin: 0;
+      padding: 0;
+      background: none;
+
+      &::backdrop {
+        background-color: transparent;
+      }
+    }
+
+    .popover-menu {
+      padding: 4px;
+      border-radius: 8px;
+      border: solid 1px ${color.divider.subtle};
+      background: ${color.container.background};
+      box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.05);
```

Comment:
hey, this shadow is slightly suspiscious - did the designer not use an existing elevation token?

Maybe `{effect.elevation[1]}`?

This probably won't look quite right in dark mode (i.e. no shadow black-on-black)